WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderFollowButton.kt

```diff
@@ -48,30 +48,17 @@ class ReaderFollowButton @JvmOverloads constructor(
             }
```

Comment:
I wonder if we should remove this `ReaderFollowButtonType` since we are making changes to this class. It looks like this "type" was used in the past to use the same component for following both sites and comments (with slightly different icon and text in the button) but I couldn't find the usage of the `FOLLOW_COMMENTS` type, so I think it has been removed?

I ask that mainly because now we are only dealing with text and using the `styles` to either keep showing the icon (temporarily for other features that use this button) or use this new button style, so it seems the `FOLLOW_COMMENTS` wouldn't work properly if it ends up being used by someone.

Wdyt?

Reply:
Good catch! Class removed.